### PR TITLE
project-todos Andrea Larsson

### PR DESCRIPTION
**Pull request by Andrealarsson**
Link to my deployed project:
https://tasks-andrea.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/src/components/Header.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import moment from 'moment'
+
+import styled from 'styled-components'
+ 
+const Header = () => {
+  const currentDate = moment().format("dddd, MMM Do");
+  const currentTime = moment().format("LT");
```
Comment (CarolineF3) [inline code/src/components/Header.js:8]: I don't think Moment is used anymore, perhaps look into a different solution for showing date and time. :)

**Review by CarolineF3 — APPROVED** [review]
This is such a lovely project! I love how much effort you've put into the styling, the list looks so neat and elegant. :)  Your code is very clean and easy to read. Great use of styled components! Excellent job! 🎉❤

File: code/src/components/AddTodo.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        </AddButton>
+      </TodoForm>
+    </>
+  )
+}
+
+export default AddTodo
+
+const AddTodoText = styled.p`
+color: #ffffff;
```
Comment (KarolCislowski) [inline code/src/components/AddTodo.js:47]: You sometimes use short sometimes long hex color notation. please stay with one convention 😸

File: code/src/components/Header.js (hunk 1 above)
Comment (KarolCislowski) [reply code/src/components/Header.js:8]: She is using it in line 14 😄

**Review by KarolCislowski — APPROVED** [review]
Nice design and grat code 🦄

**Pull request closed without merging**